src/mlpack/methods/amf/termination_policies/simple_residue_termination.hpp

```diff
@@ -82,7 +82,9 @@ class SimpleResidueTermination
     Log::Info << "Iteration " << iteration << "; residue " << residue << ".\n";
 
     // Check if termination criterion is met.
-    return (residue < minResidue || iteration > maxIterations);
+    // If maxIterations == 0, there is no iteration limit.
+    return (residue < minResidue || 
+      ((maxIterations != 0) && (iteration > maxIterations)));
```

Comment:
When wrapping a line, the next line should be tabbed twice from where the previous line began. Also we can remove the brackets around ``maxIterations != 0`` and ``iteration > maxIterations``.